packages/connect-examples/new-example/app/components/common/LogDisplay.tsx

```
@@ -0,0 +1,432 @@
+  const handleClearLogs = useCallback(() => {
+    clearLogs();
+    // 清除日志后重置过滤器，避免混乱
+    setFilter("all");
+    setSearchTerm("");
+  }, [clearLogs]);
+
+  const handleCopyLog = (log: LogEntry) => {
+    const logText = `[${log.timestamp}] [${log.type.toUpperCase()}] ${
+      log.message
+    }${log.data ? "\n" + JSON.stringify(log.data, null, 2) : ""}`;
+    navigator.clipboard.writeText(logText);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)